src/platform/silabs/wifi/wf200/WifiInterface.cpp

```diff
@@ -293,6 +293,17 @@ sl_status_t get_all_counters(void)
     return result;
 }
 
+/**
+ * @brief Convert RCPI to RSSI
+ * @param[in]  rcpi: Received Channel Power Indicator value
+ * @return RSSI value
+ */
+int16_t ConvertRcpiToRssi(uint32_t rcpi) {
+    // Checking for the overflows
+    // If rcpi is greater than or equal to INT16_MAX, return INT16_MIN to indicate an error
+    VerifyOrReturnValue(rcpi < INT16_MAX, INT16_MIN);
```

Comment:
For reference, `std::numeric_limits` is the C++ way to get the type max and min values.

Reply:
added std::numeric_limits